docs/function-file.md

```diff
@@ -41,10 +41,26 @@ Default: `sync`.
 function. If this function exceeds this limit during execution, it is stopped
 and error message is logged. Default: `128`.
 
+`headers` (optional) is a set of HTTP headers to be returned in the response of
+this function calls.
+
 `config` (optional) is a set of configurations to be passed onto the route
 setup. These configuration options shall override application configuration
 during functions execution.
 
 `build` (optional) is an array of shell calls which are used to helping building
 the image. These calls are executed before `fn` calls `docker build` and
```

Comment:
Build is used in of `fn build`. If we put `fn build` then it would become a recursive explanation.